data/items.js

```diff
@@ -6009,9 +6009,17 @@ let BattleItems = {
 			},
 		},
 		onUpdate(pokemon) {
+			if ((this.gameType === "doubles")) {
```

Comment:
This doesn't look like a good idea to me, since it's too specific, but do you even need it any more?

Reply:
probably not I was going to add some tests for singles and test without the check